src/api/app/controllers/webui/patchinfo_controller.rb

```diff
@@ -153,8 +152,14 @@ def save
           end
         end
         begin
-          frontend.put_file(xml, project: @project,
-                                 package: @package, filename: filename)
+          authorize @package, :update?
+
+          Package.verify_file!(@package, '_patchinfo', xml)
+
+          Suse::Backend.put @package.source_path('_patchinfo', user: User.current.login), xml
+
+          @package.sources_changed(wait_for_update: true) # wait for indexing for special files
```

Comment:
@adrianschroeter that line is the only thing that I'm not sure of... can you review it please?